app/templates/components/validated-input.hbs

```diff
@@ -0,0 +1,24 @@
+{{!-- Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies --}}
```

Comment:
You're right Andy - it seems to work in this context.  So now my validated-input component is working with no modifications from the original copied from the ember-cp-validations dummy app.  Hopefully Max and I can share the validated-input component without having to duplicate.  I see he made some other modifications to his.